### ISSUE TITLE
master branch is not stable currently

### ISSUE THREAD

**Issue by coezbek** [sl5net/global-IntelliSense-everywhere#7]
When downloading current master branch, GI does not currently work:

- Debug mode seems to be enabled, lots of beeping and windows.
- Following the example from the Video don't work (for example typing "__" causes !ups)

**Comment by sl5net** [comment]
Yes I'm sorry. I had a serious shoulder injury (hospital and some treatment afterwards) and a full time job. I had to commute a lot to work. Unfortunately I could not fix this coarse bug. I hope I can do this in the next few weeks.
